#!/usr/bin/env python

PACKAGE = "autonomous_bicycle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sigma_v", double_t, 0, "sigma_v", 0.15, 0.001, 0.2)
gen.add("sigma_phi_dot", double_t, 0, "sigma_phi_dot", 0.01, 0.001, 0.2)
gen.add("sigma_delta_dot", double_t, 0, "sigma_delta_dot", 0.44, 0.001, 0.6)

gen.add("R_x", double_t, 0, "R_x", 0.5, 0.01, 15.0)
gen.add("R_y", double_t, 0, "R_y", 0.5, 0.01, 15.0)
gen.add("R_z", double_t, 0, "R_z", 0.5, 0.01, 15.0)
gen.add("R_sigma", double_t, 0, "R_sigma", 0.5, 0.01, 10.0)
gen.add("R_psi", double_t, 0, "R_psi", 0.5, 0.001, 5.0)
gen.add("R_phi", double_t, 0, "R_phi", 0.8, 0.001, 5.0)

exit(gen.generate(PACKAGE, "autonomous_bicycle", "pose_estimation_interaction"))